src/webui/www/private/views/preferences.html

```diff
@@ -577,9 +577,11 @@
                     </td>
                     <td>
                         <select id="max_ratio_act">
-                            <option value="0">QBT_TR(Pause them)QBT_TR[CONTEXT=OptionsDialog]</option>
-                            <option value="1">QBT_TR(Remove them)QBT_TR[CONTEXT=OptionsDialog]</option>
-                            <option value="2">QBT_TR(Enable super seeding for them)QBT_TR[CONTEXT=OptionsDialog]</option>
+                            <option value="0">QBT_TR(Pause torrent)QBT_TR[CONTEXT=OptionsDialog]</option>
+                            <option value="1">QBT_TR(Remove torrent)QBT_TR[CONTEXT=OptionsDialog]</option>
+                            <option value="2">QBT_TR(Remove torrent + files)QBT_TR[CONTEXT=OptionsDialog]</option>
+                            <option value="3">QBT_TR(Remove torrent + folder)QBT_TR[CONTEXT=OptionsDialog]</option>
+                            <option value="4">QBT_TR(Enable super seeding for it)QBT_TR[CONTEXT=OptionsDialog]</option>
```

Comment:
We should bump the web api minor version since this value is changing from 2 to 4. So 2.2.0 -> 2.3.0

Reply:
It's not changed after all.